apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -17,10 +18,25 @@ import './SupportDetailWip.css';
 
 function Header(props: { title: string }) {
   const { title } = props;
+  const navigate = useNavigate();
   return (
     <nav className="support-header">
-      <ArrowBackIosNewIcon />
-      <Typography variant="h4">{title}</Typography>
+      <IconButton
+        aria-label="back"
+        sx={{ color: 'text.primary' }}
+        onClick={() => navigate(-1)}
```

Comment:
I think we should link to the landing page because this always goes 1 back in history regardless of what the previous page is. e.g. if i go to `google.com` then `http://localhost:3000/support-detail`, this will bring me back to `google.com`.

Reply:
Hmmm, I think in general the behavior of the back button should be returning to where the user comes from, instead of going to the Landing Page. If we prefer going back to Landing Page all the time, I think a "Home" button will be better.